R/composition.R

```diff
@@ -47,8 +47,8 @@
 #'
 #' @examples
 #' if(interactive()) {
-#'   library(shiny)
-#'   library(shiny.gosling)
+#'   require(shiny)
```

Comment:
I wonder why are we using require instead of library :thinking:

Reply:
Thanks for catching this. Reversed.